xchange-binance/src/main/java/org/knowm/xchange/binance/BinanceAuthenticated.java

```diff
@@ -0,0 +1,231 @@
+package org.knowm.xchange.binance;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.List;
+
+import javax.ws.rs.DELETE;
+import javax.ws.rs.FormParam;
+import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+
+import org.knowm.xchange.binance.dto.BinanceException;
+import org.knowm.xchange.binance.dto.account.BinanceAccountInformation;
+import org.knowm.xchange.binance.dto.trade.BinanceCancelledOrder;
+import org.knowm.xchange.binance.dto.trade.BinanceNewOrder;
+import org.knowm.xchange.binance.dto.trade.BinanceOrder;
+import org.knowm.xchange.binance.dto.trade.BinanceTrade;
+import org.knowm.xchange.binance.dto.trade.OrderSide;
+import org.knowm.xchange.binance.dto.trade.OrderType;
+import org.knowm.xchange.binance.dto.trade.TimeInForce;
+
+import si.mazi.rescu.ParamsDigest;
+
+@Path("")
+@Produces(MediaType.APPLICATION_JSON)
+public interface BinanceAuthenticated extends Binance {
+    
+    public static final String SIGNATURE = "signature";
+    static final String X_MBX_APIKEY = "X-MBX-APIKEY";
+
+    @POST
+    @Path("api/v3/order")
+    /**
+     * Send in a new order
+     * @param symbol
+     * @param side
+     * @param type
+     * @param timeInForce
+     * @param quantity
+     * @param price     optional, must be provided for limit orders only
+     * @param newClientOrderId optional, a unique id for the order. Automatically generated if not sent.
+     * @param stopPrice optional, used with stop orders
+     * @param icebergQty optional, used with iceberg orders
+     * @param recvWindow        optional
+     * @param timestamp
+     * @return
+     * @throws IOException
+     * @throws BinanceException
+     */
+    BinanceNewOrder newOrder(@FormParam("symbol") String symbol
```

Comment:
Do you have the Eclipse or IntelliJ formatter that makes the params get formatted like this? I like it.

Reply:
i use eclipse, but this is my own preferred formatting